src/app/hierarchical-grid/hierarchical-grid-row-reorder/hierarchical-grid-row-reorder.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, ViewChild } from "@angular/core";
+import { IgxDropEventArgs, IgxHierarchicalGridComponent, IgxHierarchicalRowComponent } from "igniteui-angular";
+import { createData, IDrive } from "../../data/files.data";
+
+interface ICursorPosition {
+    x: number;
+    y: number;
+}
+
+@Component({
+    selector: "hierarchical-grid-row-reorder",
+    styleUrls: ["./hierarchical-grid-row-reorder.component.scss"],
```

Comment:
Please specify some minor alignment styles in the styles files, as the sample is not properly centered when viewed in **docfx**
For example:
```
:host {
	display: flex;
    flex-flow: row;
    align-items: center;
    justify-content: center;
    padding: 8px 0px;
}
```